python-sdk/example_dags/example_load_file.py

```diff
@@ -215,7 +214,7 @@
     # [END load_file_example_17]
 
     # [START load_file_example_18]
-    dataframe = aql.load_file(input_file=File(path=data_url, filetype=FileType.CSV))
+    dataframe = aql.load_file(input_file=File(path="s3://astro-sdk/sample_csv.data", filetype=FileType.CSV))
```

Comment:
what if a user wants to use this file ? It is not accessible unless thats a public s3 bucket.

Reply:
can we store it in `python_sdk/example_dags/data` folder?

Reply:
But the case we are handling as part of this example is that the file is of a different extension than `csv` but the filetype is `csv`.

Reply:
Changed the folder location as suggested here: https://github.com/astronomer/astro-sdk/pull/1297/commits/91eab7b161577a51bcc0445c17cdcf9a2d96f6ab

Reply:
I can create a separate issue for the following

> I am more concerned about the fact that we are making the data not available to users who want to try out this DAG for the first time

Reply:
Created a issue for this https://github.com/astronomer/astro-sdk/issues/1299
@phanikumv